firebase-sessions/firebase-sessions.gradle.kts

```diff
@@ -51,6 +51,8 @@ dependencies {
 
   runtimeOnly("com.google.firebase:firebase-installations:17.1.3")
 
+  implementation("com.google.android.datatransport:transport-api:3.0.0")
+
```

Comment:
I also see the following in other SDKs. Do I need them?

```

   implementation("com.google.android.datatransport:transport-runtime:3.1.8")
   implementation("com.google.android.datatransport:transport-backend-cct:3.1.8")
```

Reply:
I think you don't need `com.google.android.datatransport:transport-backend-cct`. Basically, there is an API component to use the APIs, runtime component to do the actual operation and backend-cct is for clearcut compatibility. We don't need CCT compatibility here.

Here is how Fireperf depends on Firelog. https://github.com/firebase/firebase-android-sdk/blob/master/firebase-perf/firebase-perf.gradle#L104-L105

So either a runtime dependency on their runtime dependency or a pointed dependency to a production version - both are fine.